src/EnergyPlus/MixedAir.cc

```diff
@@ -565,7 +563,6 @@ void SimOAComponent(EnergyPlusData &state,
         OACoolingCoil = true;
     } else if (CompTypeNum == SimAirServingZones::CompType::DXSystem) { // CoilSystem:Cooling:DX  old 'AirLoopHVAC:UnitaryCoolOnly'
```

Comment:
@rraustad Do we have a regression test file that uses CoilSystem:Cooling:DX in an AirloopHVAC:OutdoorAirSystem?

Reply:
I have only found water coils in the OA system. I could create one using an existing example file. I also need a test file for the faults model as well.

Reply:
There is something strange happening. I added CoilSystem:Cooling:DX to OA System in FurnaceWithDXSystemRHControl. In this file there is a UnitarySystem in the air loop, and I added CoilSystem to the OA system. That's a good test of multiple objects in the same file.

Existing:

    Branch,
      Air Loop Main Branch,    !- Name
      ,                        !- Pressure Drop Curve Name
      AirLoopHVAC:OutdoorAirSystem,  !- Component 1 Object Type
      OA Sys 1,                !- Component 1 Name
      Outdoor Air Mixer Inlet Node,  !- Component 1 Inlet Node Name
      Mixed Air Node,          !- Component 1 Outlet Node Name
      AirLoopHVAC:UnitarySystem,  !- Component 2 Object Type
      GasHeat DXAC Furnace 1,  !- Component 2 Name
      Mixed Air Node,          !- Component 2 Inlet Node Name
      Air Loop Outlet Node;    !- Component 2 Outlet Node Name

New CoilSystem added to OA system:

    AirLoopHVAC:OutdoorAirSystem:EquipmentList,
      OA Sys 1 Equipment,      !- Name
      CoilSystem:Cooling:DX,   !- Component 1 Object Type
      OA DX Cooling Coil,      !- Component 1 Name
      OutdoorAir:Mixer,        !- Component 2 Object Type
      OA Mixing Box 1;         !- Component 2 Name

Sim is crashing on this->AirInNode = 0 here.

    Real64 tempMassFlowRateMaxAvail = state.dataLoopNodes->Node(this->AirInNode).MassFlowRateMaxAvail;

The thing is I see it setting that node num during getInput. I wonder if this is like report vars that this needs to be set once all systems are gotten. Still looking at this one.

![image](https://user-images.githubusercontent.com/5778156/125560796-b9109095-241d-4e77-a0a6-8f57ca725bc0.png)